### ISSUE TITLE
Fix travis tests on newer linux distros

### ISSUE THREAD

**Issue by siddharthab** [grailbio/bazel-toolchain#31]
Looks like the builds are failing with the message:
```
_toolchain/bin/clang: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
```

See https://travis-ci.org/grailbio/bazel-toolchain/jobs/604832406.

**Comment by terinjokes** [comment]
I'm not seeing this with LLVM 8.0.0, but it does appear with LLVM 9.0.0. Looks like the latter was compiled with `LLVM_ENABLE_TERMINFO`. It's not clear if this was an intentional change.

```console
$ ldd /home/terin/.cache/bazel/_bazel_terin/b7a8356babac270126021e7c3f7082e2/external/llvm_toolchain/bin/clang-9
	linux-vdso.so.1 (0x00007ffee8354000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007fc26eb9e000)
	libz.so.1 => /usr/lib/libz.so.1 (0x00007fc26eb84000)
	librt.so.1 => /usr/lib/librt.so.1 (0x00007fc26eb79000)
	libdl.so.2 => /usr/lib/libdl.so.2 (0x00007fc26eb74000)
	libtinfo.so.5 => not found
	libm.so.6 => /usr/lib/libm.so.6 (0x00007fc26ea2e000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007fc26e842000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007fc26e828000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007fc26e661000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007fc26ebfd000)
```

Installing `ncurses5-compat-libs` on Arch works, but I'm reverting to 8.0.0 for now.

**Comment by siddharthab** [comment]
Thanks @terinjokes for the research.

Some more background on this: https://reviews.llvm.org/D42274#1076895

I will add the dep to  the docker images.

**Comment by siddharthab** [comment]
Instead of adding the deps, I regressed our tested version to be 8.0.0 (016b4256551c89e911bdf9762ea9caf0b8abf5ab) and fixed libtinfo5 on the debian image (e7709c3df9f34e9ddee62993c90363067ea6771b).
